frontend/packages/ceph-storage-plugin/integration-tests/tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,231 @@
+import { execSync } from 'child_process';
+import { ExpectedConditions as until, browser } from 'protractor';
+import * as crudView from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  goToOperatorHub,
+  installOperator,
+  InstallCluster,
+  searchInOperatorHub,
+  uninstallOperator,
+  storageClusterLink,
+  btnCreate,
+  selectWorkerRows,
+  status,
+  operatorsPage,
+  ocsOperator,
+  ocsOperatorStatus,
+  filterInput,
+  getDataFromRowAndCol,
+  podNameFilter,
+  goToStorageClasses,
+  checkIfClusterIsReady,
+  getPodData,
+  getPodPhase,
+  getPodName,
+  waitFor,
+  testPodIsRunning,
+} from '../views/subscriptionFlow.view';
+import { NS, SECOND, MINUTE } from '../utils/consts';
+import { ocsDeploy } from '../utils/mocks';
+
```

Comment:
Please make sure the script implements the steps and their expected behaviors by referring to https://docs.google.com/spreadsheets/d/16__6G4dfPYFbvIIRmSCgjzzHSwXF2cPYVYON6GRL7ZA/edit#gid=794221253&range=B24
The above sheet has the test case for installation. 

Please contact me or Prasad(tdesala@redhat.com) for any clarification on this.